Reneiski_Homework1/ReneiskiSelenium11/PageObjects/Pages/RegFormPage.cs

```diff
@@ -0,0 +1,32 @@
+﻿using OpenQA.Selenium;
+using ReneiskiSelenium11.Common.WebElements;
+using ReneiskiSelenium11.Data;
+
+namespace ReneiskiSelenium11.PageObjects.Pages
+{
+    public class RegFormPage : WebTablePage
+    {
+        private MyWebElement FirstNameCell = new MyWebElement(By.XPath("//*[@id='firstName']"));
```

Comment:
It's actually TextBox, not cell (please correct here and below)

Reply:
Fixed